site/en/docs/extensions/reference/userscripts/index.md

```diff
@@ -0,0 +1,102 @@
+---
+api: userScripts
+---
+{% Aside 'note' %}
+This API is currently available in Chrome 120 beta.
+{% endAside %}
+## Manifest
+
+To use the `chrome.userScripts` API, add the `"userScripts"` permission to your manifest.json and `"host_permissions"` for sites you want to run scripts on.
+
+```json/9-15
+{
+  "name": "User script test extension",
+  "manifest_version": 3,
+  "minimum_chrome_version": "118",
+  "permissions": [
+    "userScripts"
+  ],
+  "host_permissions": [
+    "*://requested.com/*"
+  ]
+}
+```
+
+## Concepts and usage
+
+A user script is a bit of code injected into a web page to modify its appearance or behavior. Scripts are either created by users or downloaded from a script repository using a type of extension called a user script manager.
```

Comment:
I find the phrasing "using a type of extension called a user script manager." a bit weird. Can't this be done by any extension? It sounds like a user script manager is a special kind of extension. Or am I missing something? //cc @oliverdunk

Reply:
I think that's fair feedback. I see what this is trying to say but it might be worth clarifying that user script managers are just a category of extension people build vs. an actual type that is somehow registered differently. Although I'm not sure what words you would need for that 😅

Reply:
I changed the ending to 'a script repository or a user script extension.'